scripts/cleanup_migrations.py

```diff
@@ -18,24 +18,14 @@ def main():
     # Get absolute path of the repository's root from the script location
     root_folder = os.path.abspath(os.path.join(
         os.path.dirname(__file__), os.pardir))
-    config_file = os.path.join(root_folder, 'config', 'config')
-
-    if not os.path.exists(config_file):
-        print('ERROR: config file is missing')
-        sys.exit(1)
-
-    # Try importing the mozilla-unified path from config/config
-    with open(config_file, 'r') as cfg:
-        lines = cfg.readlines()
-        for line in lines:
-            if line.startswith("mozilla_unified_path"):
-                mozilla_unified_path = line.split('=')[1].strip().strip('"')
 
+    # Read paths from config file
     try:
-        mozilla_unified_path
-    except NameError:
-        print('mozilla_unified_path is not defined in the config file')
-        sys.exit(1)
+        [mozilla_unified_path] = local_config.read_config(
+            ['mozilla_unified_path'])
+    except Exception as e:
+        print('Error reading paths from config')
+        print(e)
```

Comment:
I'd drop the exception handling, `local_config` is designed to sys.exit on errors.

Everything else can be just stack traces, too.

Or, design `local_config` to actually raise Expressions on errors, and then you could `sys.exit(e)` here.

Reply:
I think I'll keep it simple and remove the exception handling completely.